src/components/bold.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Bold as BoldPrimitive } from "./bold";
+import argTypes from "./bold.props.json"
+
+export default {
+  title: "Components/Bold",
+  component: BoldPrimitive,
+  argTypes: {},
+} as ComponentMeta<typeof BoldPrimitive>;
+
+const Template: ComponentStory<typeof BoldPrimitive> = (args) => (
+  <BoldPrimitive {...args} />
+);
+
+export const Bold = Template.bind({});
```

Comment:
Note to self: we can rewrite these to just exporting a component that renders the primitive instead of using .args, right?

Reply:
right, forgot about it, we need them